Lib/test/test_traceback.py

```diff
@@ -1555,6 +1555,14 @@ def __repr__(self):
         err_msg = '<note str() failed>'
         self.assertEqual(self.get_report(e), vanilla + err_msg + '\nFinal Note\n')
 
+        e.__notes__  = "please do not explode me"
```

Comment:
I wonder if we should have a notes setter that fails when you set something like this.

Reply:
I guess not, since we tolerate all sorts of things here.